packages/mad-tag-ocr/src/index.ts

```diff
@@ -0,0 +1 @@
+export { OCRCamera } from "./components/Camera/OCRCamera";
```

Comment:
Maybe we should also export the ORCCameraProps to give users some flexibility for how they want to set up the camera